src/methods/modular_programs/easy/q3/MeanNMedian.java

```diff
@@ -0,0 +1,34 @@
+package methods.modular_programs.easy.q3;
+
+/*
+Write a program that takes three initialized and declared integer variables and determines their mean and median.
+Note that the mean is the average of the three numbers and that the median is the middle number when the numbers are
+sorted from least to greatest. The program should have separate methods for computing both of these values, one called
+meanCalc and the other called medianCalc.
```

Comment:
Add this to the end of the question:
`Finally, the program should output the median and mean in the main method. `